keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -941,7 +941,12 @@ kpxc.fillFromPopup = async function(id, uuid) {
     }
 
     await sendMessage('page_set_login_id', id);
-    kpxc.fillInCredentials(kpxc.combinations[0], kpxc.credentials[id].login, uuid);
+    const selectedCredentials = kpxc.credentials.find(c => c.uuid === uuid);
+    if (!selectedCredentials) {
+        return;
```

Comment:
Can we issue a console warning here? Does this warrant a user level warning as well?

Reply:
A console warning should be enough.